### DISCUSSION TITLE
CPU Temperature Glances and InfluxDB2

### DISCUSSION THREAD

**Discussion by felipebraz** [nicolargo/glances#2840] [Q&A] [answered]
Hi,

I've configured Glances to export to InfluxDB2 and the data is going just fine, but I couldn't find the CPU Temperature on the CPU data.
On the web view the data is there.

Do I need to set something besides the usual?

Followed this configuration: https://glances.readthedocs.io/en/latest/gw/influxdb.html#influxdb-v2-from-influxdb-v1-8-x-flux-and-influxdb-v2-x

Config File:

[global]
refresh=2
check_update=false
history_size=28800
[influxdb2]
host=1.2.3.4
port=8086
protocol=http
org=0xxxx5456xxx
bucket=Proxmox
token=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Service File:

[Unit]
Description=Glances Exporter
After=network.target

[Service]
ExecStart=/usr/local/bin/glances -t 5 --export influxdb2 --quiet
Restart=on-abort
RemainAfterExit=yes

[Install]
WantedBy=multi-user.target

**Comment by nicolargo** [comment, marked as answer]
Hi @felipebraz 

the latest Docker images (4.1.1) should solve the issue: https://github.com/nicolargo/glances/issues/2856

**Reply by felipebraz** [reply]
I'm not using Docker, I installed directly on Proxmox OS, but I'll try to update it (as soon as I learn how LoL) and test.
Thanks!

**Reply by SConaway** [reply]
Latest PyPi release is fixed for me. I'm on debian and ran `pip3 install --user --upgrade glances[all]` and it's working again (after restarting my custom `systemd` unit).

**Reply by felipebraz** [reply]
Worked! Thank you very much!
